package.json

```diff
@@ -689,7 +689,7 @@
         "@types/mocha": "^2.2.42",
         "@types/node": "^14.14.33",
         "@types/require-from-string": "^1.2.0",
-        "@types/vscode": "1.50.0",
+        "@types/vscode": "^1.56.0",
```

Comment:
Please stick to 1.56.0 here. (Remove `^`)

Reply:
Agreed.